packages/private-build-infra/src/stripped-build-plugins.js

```diff
@@ -23,78 +19,22 @@ function isProduction(environment) {
   return /production/.test(environment);
 }
 
-module.exports = function(environment, isLocalBuild) {
-  let flagsJsonPath = __dirname + '/../config/in-progress-features.json';
-  let flagsJson = fs.readFileSync(flagsJsonPath, { encoding: 'utf8' });
-  let inProgressFlags = JSON.parse(flagsJson);
+module.exports = function(environment) {
+  let plugins = [];
+  const DebugMacros = require('./debug-macros')(environment);
   let filteredImports = {};
-  let allowInProgressFeatures = isLocalBuild === true && wantsEnabledFeatures();
-  let manuallyEnabled = isLocalBuild ? getManuallyEnabledFeatures() : {};
-  let enabledFlags = [];
-  let allFlags = Object.assign({}, inProgressFlags, manuallyEnabled);
-
-  for (let flag in allFlags) {
-    let state = inProgressFlags[flag];
-    /*
-     Default anything not `true` (e.g. `null`) feature to `false`
-     unless this is a local build and we've set the flags
-     to enable something.
-    */
-    if (manuallyEnabled[flag]) {
-      if (state === true) {
-        // eslint-disable-next-line no-console
-        console.warn('You specified the in-progress-feature "' + flag + '" but it was already active');
-      } else if (state === undefined) {
-        throw new Error('You specified the in-progress-feature "' + flag + '" but no such flag exists!');
-      } else {
-        // eslint-disable-next-line no-console
-        console.warn('Manually Actived in-progress-feature: ' + flag);
-      }
-      inProgressFlags[flag] = true;
-    } else if (allowInProgressFeatures && state === null) {
-      enabledFlags.push(flag);
-      inProgressFlags[flag] = true;
-    } else if (state !== true) {
-      inProgressFlags[flag] = false;
-    }
-  }
-
-  if (allowInProgressFeatures) {
-    if (enabledFlags.length) {
-      // eslint-disable-next-line no-console
-      console.warn(
-        'Enabled the following in-progress-features that specified `null` as their state for this build: ["' +
-          enabledFlags.join(', ') +
-          '"]'
-      );
-    } else {
-      // eslint-disable-next-line no-console
-      console.warn(
-        'Attempted to enable all in-progress-features that specify `null` as their state for this build, but there were none.'
-      );
-    }
-  }
-
   let postTransformPlugins = [];
-  let plugins = [
-    [
-      FeatureFlags,
-      {
-        import: { module: 'ember-data/-private/features' },
-        features: inProgressFlags,
-      },
-    ],
-  ];
 
-  if (isProduction(environment) || isInstrumentedBuild()) {
+  if (isProduction(environment)) {
     postTransformPlugins.push([StripClassCallCheck]);
     uniqueAdd(filteredImports, '@ember-data/store/-debug', ['assertPolymorphicType']);
   }
 
   plugins.push(
     [FilterImports, { imports: filteredImports }],
     [StripFilteredImports, filteredImports],
-    [TransformBlockScoping, { throwIfClosureRequired: true }]
+    [TransformBlockScoping, { throwIfClosureRequired: true }],
+    ...DebugMacros
```

Comment:
This got added in this PR?

Reply:
No, just moved to the appropriate spot. When the new infra got added before we added it somewhere else instead of here.